climpred/metrics.py

```diff
@@ -143,6 +252,20 @@ def _spearman_r(forecast, reference, dim=None, **metric_kwargs):
     return spearman_r(forecast, reference, dim=dim, weights=weights, skipna=skipna)
 
 
+_spearman_r = Metric(
+    name='spearman_r',
+    longname="Spearman's Anomaly correlation coefficient",
+    function=_spearman_r,
+    is_positive=True,
+    is_probabilistic=False,
+    aliases=['sacc'],
+    unit_power=0.0,
+    min=-1.0,
+    max=1.0,
+    perfect=1.0,
+)
+
+
 def _spearman_r_p_value(forecast, reference, dim=None, **metric_kwargs):
     """
     Calculate the probability associated with the ACC not being random.
```

Comment:
This should be similar to the Pearson version: "Calculate the probability associated with Person's Anomaly Correlation". Maybe just "p-value associated with Spearman's anomaly correlation coefficient." and the same for pearson p.